unit_tests/test_Utils.py

```diff
@@ -304,7 +304,14 @@ def test_toDatetime(self):
         self.assertEqual(
             datetime.datetime(2010, 1, 1, 23, 59, 59, 999999),
             Utils.toDatetime(datetime.date(2010, 1, 1), end=True))
-
-
+    def test_load_source(self):
+        """Testing load_source in Utils.py"""
+        filename = "temp_testfile.py"
+        with open(filename, "w") as f:
+            f.write("def hello():\n    return 'Hello, world!'\n")
+        inspect = None
+        module = Utils.load_source("temp_testfile", filename, inspect)
+        self.assertEqual(module.hello(), 'Hello, world!')
+        
```

Comment:
Nice test. Do leave a blank line before it, and two after (because it's the end of the class). Use tempfile to make a temporary directory and put your test file in there, then clean it up after (with a try/finally)--that way you're not leaving it in the current directory.